content/backstage/plugins/buildkite.md

```diff
@@ -40,30 +34,39 @@ gettingStarted:
             Authorization: 
               $env: BUILDKITE_TOKEN
 
+  - intro: Add plugin to the list of your plugins
+    language: typescript
```

Comment:
This step is no longer necessary AFAIK

Reply:
Ah ok cool! I had just taken it from here: https://github.com/RoadieHQ/backstage-plugin-buildkite/blob/main/README.md. I'll remove it now so.